components/org.wso2.carbon.identity.user.endpoint/src/test/java/org.wso2.carbon.identity.user.endpoint.Util/HandleExceptionsTest.java

```diff
@@ -65,6 +67,33 @@ public void testConflictException() {
         assertEquals(conflictException.getResponse().getStatus(), 409);
     }
 
+    @Test
+    public void testHandleConflictException(){
 
-}
+        ConflictException conflictException = new ConflictException();
+        assertEquals(conflictException.getResponse().getStatus(),409);
+        String errorCode = "UNR-10002";
```

Comment:
Aren't there any constants for error codes

Reply:
These are just test codes. Will change the value to reflect it.